CHANGELOG.md

```diff
@@ -2,7 +2,54 @@
 
 ## [Unreleased](https://github.com/CosmWasm/wasmd/tree/HEAD)
 
-[Full Changelog](https://github.com/CosmWasm/wasmd/compare/v0.32.0...HEAD)
+[Full Changelog](https://github.com/CosmWasm/wasmd/compare/v0.40.0...HEAD)
+
+## [v0.40.0](https://github.com/CosmWasm/wasmd/tree/v0.40.0) (2023-05-2x)
+
+[Full Changelog](https://github.com/CosmWasm/wasmd/compare/v0.32.0...v0.40.0)
+
```

Comment:
We could add
`- Fix cli update-instantiate-config command [\#1415](https://github.com/CosmWasm/wasmd/pull/1415)`